packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -147,7 +150,11 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     [onFetchAvatarPersonaData]
   );
 
-  const messageThreadStyles = Object.assign({}, messageThreadChatCompositeStyles, styles?.messageThread);
+  const messageThreadStyles = Object.assign(
+    {},
```

Comment:
Do we need this empty object here incase everything else is undefined? I haven't seen this way of defining the styles on a component.